File: sdk/src/main/java/com/meniga/sdk/webservices/requests/GetBudgetEntryById.java
package com.meniga.sdk.webservices.requests;

/**
 * Copyright 2017 Meniga Iceland Inc.
 */
public class GetBudgetEntryById extends QueryRequestObject {

	public long budgetId;
	public long entryId;

	@Override
	public long getValueHash() {
		int result = (int) (budgetId ^ (budgetId >>> 32));
		result = 31 * result + (int) (entryId ^ (entryId >>> 32));
		return result;
	}
}

File: sdk/src/main/java/com/meniga/sdk/webservices/budget/GetBudgetEntryById.kt
/*
 * Copyright 2017 Meniga Iceland Inc.
 */
package com.meniga.sdk.webservices.budget

import com.meniga.sdk.webservices.requests.QueryRequestObject

internal data class GetBudgetEntryById(
        var budgetId: Long = 0,
        var entryId: Long = 0
) : QueryRequestObject() {

    override fun getValueHash(): Long = hashCode().toLong()
}
